core/domain/email_manager.py

```diff
@@ -61,34 +61,44 @@
 PlatformParameterRegistry = platform_parameter_registry.Registry
 
 
-NEW_REVIEWER_EMAIL_DATA: Dict[str, Dict[str, str]] = {
+NEW_CONTRIBUTOR_EMAIL_DATA: Dict[str, Dict[str, str]] = {
```

Comment:
Hi @laeeraa, sorry for not seeing this earlier, but I realized that there is an ambiguity in the chosen naming. We usually think of "contributors" as the folks who submit translations/questions, and their submissions are reviewed by reviewers. You are correct that, in a way, all of these folks are "contributors" because they use the contributor dashboard, but the existing association of the term "contributors" to the submitters makes this a bit confusing. So every time I read the code, I find myself having to step back and realize "contributors means everyone here, not just submitters" and I think this could cause confusion down the line too.

Would you be open to changing this to NEW_CD_USER_EMAIL_DATA, and adjusting similar nomenclature elsewhere (i.e. to CD_USER for cases which refer to both submitters and reviewers)? Here, "CD" means contributor dashboard. It is a bit awkward but I can't think of a better name that avoids the connotation issues I mentioned above.

Also, if it makes anything easier, feel free to drop any code relating to voiceover submissions. We don't currently support that, and aren't planning to do so going forward anyway. This is optional though, I only mention it in case it helps with figuring out the conceptual naming. 

Thanks!

Reply:
Ah okay! Sure, I will change it within the next two days:)

Reply:
![grafik](https://github.com/oppia/oppia/assets/92863142/ecc54eb7-dbfa-4acf-9b78-3337f9c66035)

I would also rename those constants in assets/constants.ts to: 

![grafik](https://github.com/oppia/oppia/assets/92863142/b8b4d16d-f201-4c6d-ac5c-6c13647cde73)

@seanlip would this be fine with you?

Reply:
done with: 
https://github.com/oppia/oppia/blob/4858ea094646a84f8cc31a5d323a1e9e58ca331b/assets/constants.ts#L6020-L6029